cypress/component/DataAccessRequest/researcher_info_new.spec.js

```diff
@@ -65,6 +66,17 @@ const WrappedResearcherInfo = (props) => {
   </BrowserRouter>;
 };
 
+const user = {
+  userId: 1,
+  dacUserId: 2,
+  displayName: 'Cindy Crawford',
+  email: 'cc@c.com'
+};
+
+beforeEach(() => {
+  cy.stub(User, 'getMe').returns(Promise.resolve(user));
```

Comment:
Can you just returns(user) or does something require a Promise.resolve?

Reply:
we worked through a bug where it had to return a promise, since it was `await`ing it.

Reply:
Do you remember where that was in the code?  I'd like to see that.

It looks like User.getMe returns response.data. In that setup, the request will already have been resolved and the data property of the response is returned.  Could something else be getting in the way?